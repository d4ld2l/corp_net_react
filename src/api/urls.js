// @flow

export const API_URL = '/api'
export const APP_SETTINGS = `${API_URL}/settings`
export const BIRTHDATES_NEAREST_URL = `${API_URL}/birth_dates/nearest`
export const CANDIDATES_URL = `${API_URL}/candidates`
export const CANDIDATE_URL = `${CANDIDATES_URL}/{{ id }}`
export const CANDIDATE_VACANCIES = `${API_URL}/candidate_vacancies/{{ id }}/comments`
export const ACCOUNTS_URL = `${API_URL}/accounts`
export const GET_TICKET_URL = `${ACCOUNTS_URL}/get_ticket`
export const PROFILE_URL = `${ACCOUNTS_URL}/{{ id }}`
export const PROFILE_PHOTOS_URL = `${ACCOUNTS_URL}/{{ accountId }}/account_photos`
export const PROFILE_PHOTO_URL = `${ACCOUNTS_URL}/{{ accountId }}/account_photos/ {{ id }}`
export const CONFIRM_SKILL_URL = `${API_URL}/account_skills/{{ id }}/confirm`
export const UNCONFIRM_SKILL_URL = `${API_URL}/account_skills/{{ id }}/unconfirm`
export const EVENTS_URL = `${API_URL}/events`
export const DICTIONARIS_EVENT_TYPE_URL = `${API_URL}/dictionaries/event_type`
export const DICTIONARIES_ACCOUNTS_URL = `${ACCOUNTS_URL}/dictionary`
export const DICTIONARIES_PROJECTS_URL = `${API_URL}/projects/dictionary`
export const DICTIONARIES_VACANCIES_URL = `${API_URL}/vacancies/dictionary`
export const DICTIONARIES_VACANCY_STAGE_GROUP_URL = `${API_URL}/dictionaries/vacancy_stage_group`
export const DICTIONARIES_NEWS_CATEGORY_URL = `${API_URL}/dictionaries/news_category`

export const DICTIONARIES_DEPARTMENTS_URL = `${API_URL}/dictionaries/department`
export const DICTIONARIES_DEPARTMENTS_NAMES_URL = `${API_URL}/dictionaries/department_names`
export const DICTIONARIES_PROFESSIONAL_SPECIALIZATIONS_URL = `${API_URL}/dictionaries/professional_specialization`
export const DICTIONARIES_CANDIDATE_SKILLS_URL = `${API_URL}/dictionaries/candidate_skill`

export const DICTIONARIES_TAG_URL = `${API_URL}/dictionaries/tag`
export const DICTIONARIES_RESUME_SOURCES_URL = `${API_URL}/dictionaries/resume_source`
export const DICTIONARIES_LANGUAGES_URL = `${API_URL}/dictionaries/language`
export const DICTIONARIES_LANGUAGES_LEVEL_URL = `${API_URL}/dictionaries/language_level`
export const DICTIONARIES_EDUCATION_LEVEL_URL = `${API_URL}/dictionaries/education_level`
export const DICTIONARIES_SPECIALIZATION_URL = `${API_URL}/dictionaries/professional_area`
export const DICTIONARIES_OFFICES_URL = `${API_URL}/dictionaries/office`
export const DICTIONARIES_POSITIONS_URL = `${API_URL}/dictionaries/position`
export const DICTIONARIES_SKILLS_URL = `${API_URL}/dictionaries/skill`
export const DICTIONARIES_CITIES_URL = `${API_URL}/dictionaries/accounts_city`
export const DICTIONARIES_STRUCTURE_UNITS_URL = `${API_URL}/dictionaries/structure_unit`
export const DICTIONARIES_CONTACT_TYPES_URL = `${API_URL}/dictionaries/contract_type`
export const DICTIONARIES_EMPLOYEE_STATES_URL = `${API_URL}/dictionaries/employee_state`
export const EVENT_URL = `${EVENTS_URL}/{{ id }}`
export const EVENT_PARTICIPANTS = `${API_URL}/event_participants`
export const DICTIONARIES_SEARCH_SKILLS_URL = `${API_URL}/skills`
export const TEMPLATE_STAGES_URL = `${API_URL}/template_stages`
export const NEWS_URL = `${API_URL}/news`
export const NEW_URL = `${NEWS_URL}/{{ id }}`
export const PROFILE_ME_URL = `${ACCOUNTS_URL}/me`
export const PERMISSIONS_MY_URL = `${API_URL}/permissions/my`
export const SURVEYS_URL = `${API_URL}/surveys`
export const SURVEY_URL = `${SURVEYS_URL}/{{ id }}`
export const SURVEY_RESULTS_URL = `${API_URL}/survey_results`
export const VACANCIES_URL = `${API_URL}/vacancies`
export const VACANCY_URL = `${API_URL}/vacancies/{{ id }}`
export const VACANCY_STAGES_URL = `${API_URL}/vacancies/{{ id }}/vacancy_stages`
export const VACANCY_CANDIDATES_URL = `${API_URL}/vacancies/{{ id }}/candidates`
export const VACANCY_CANDIDATE_URL = `${API_URL}/vacancies/{{ vacancyId }}/candidates/{{ candidateId }}`
export const TRANSFER_CANDIDATE_URL = `${API_URL}/vacancies/{{ vacancyId }}/candidates/transfer`
export const EMAIL_NOTIFICATION = `${API_URL}/send_resume_letters`
export const VACANCY_CHANGE_STATE_URL = `${VACANCY_URL}/change_state/{{ state }}`
export const DEPARTMENTS_TREE_URL = `${API_URL}/departments/tree`
export const DEPARTMENTS_URL = `${API_URL}/departments`
export const SERVICES_URL = `${API_URL}/services`
export const SERVICE_URL = `${API_URL}/services/{{ id }}`
export const BIDS_URL = `${API_URL}/bids`
export const BIDS_AUTHOR_URL = `${BIDS_URL}/author`
export const BIDS_EXECUTOR_URL = `${BIDS_URL}/executor`
export const BID_URL = `${BIDS_URL}/{{ id }}`
export const BID_STATES_URL = `${BIDS_URL}/{{ id }}/states`
export const BID_ALLOWED_STATES_URL = `${BIDS_URL}/{{ id }}/allowed_states`
export const BID_CHANGE_STATE_URL = `${BIDS_URL}/{{ id }}/change_state/{{ state }}`
export const BID_STB_PARTICIPANTS_STATE_URL = `${BIDS_URL}/{{ id }}/team_building_information_accounts`
export const CUSTOMERS_URL = `${API_URL}/customers`
export const CUSTOMERS_CONTACTS_URL = `${API_URL}/customers/{{ customerId }}/contacts`
export const CUSTOMERS_CONTACT_URL = `${API_URL}/customers/{{ customerId }}/contacts/{{ id }}`
export const COMMENT_CUSTOMERS_CONTACT_URL = `${API_URL}/customers/{{ customerId }}/contacts/{{ id }}/add_comment`
export const LEGAL_UNITS_URL = `${API_URL}/legal_units`
export const LEGAL_UNITS_DICTIONARY_URL = `${API_URL}/legal_units/dictionary`
export const PROJECTS_URL = `${API_URL}/projects`
export const PROJECTS_DICRTIONARY_URL = `${API_URL}/dictionaries/tag?entity=project`
export const PROJECT_URL = `${PROJECTS_URL}/{{ id }}`
export const PROFILES_PROJECT_URL = `${PROJECTS_URL}/{{ id }}/account_projects`
export const PROFILE_PROJECT_URL = `${PROJECTS_URL}/{{ id }}/account_projects/{{project_id}}`
export const FEEDS_URL = `${API_URL}/feed`
export const USERS_COUNTER_URL = `${API_URL}/visits/stats`
export const DISTRIBUTIONS_URL = `${API_URL}/mailing_lists`
export const DISTRIBUTION_URL = `${API_URL}/mailing_lists/{{ id }}`
export const COMPARISON_SAVE_BOTH = `${API_URL}/save_both`
export const COMPARISON_SAVE_ONE = `${API_URL}/save_one`
export const TASKS_URL = `${API_URL}/tasks`
export const TASK_URL = `${API_URL}/tasks/{{ id }}`
export const SUBTASKS_URL = `${API_URL}/tasks/{{ task_id }}/subtasks`
export const SUBTASK_URL = `${API_URL}/tasks/{{ task_id }}/subtasks/{{ id }}`
export const UPDATE_PASSWORD_URL = `${ACCOUNTS_URL}/update_password`
export const VACANCY_STAGES_GROUPS = `${API_URL}/vacancy_stage_groups/stats`
export const FILTERED_ANALYTICS_STAGES_GROUPS = `${API_URL}/vacancy_stage_groups/recruitment_vortex`
export const PROFILE_HR_URL = `${ACCOUNTS_URL}/{{ id }}?hr=true`
export const DISCUSSIONABLE_URL = `discussions`
export const DISCUSSIONS_URL = `${API_URL}/${DISCUSSIONABLE_URL}`
export const DISCUSSION_URL = `${API_URL}/${DISCUSSIONABLE_URL}/{{ id }}`
export const CANDIDATE_RATINGS_URL = `${API_URL}/candidate_ratings`
export const ASSESSMENT_SESSIONS_URL = `${API_URL}/assessment/sessions`
export const ASSESSMENT_SESSION_URL = `${API_URL}/assessment/sessions/{{id}}`
export const EVALUATIONS_ASSESSMENT_SESSION_URL = `${API_URL}/assessment/sessions/{{id}}/evaluations`
export const ASSESSMENT_SESSION_RESULT_URL = `${API_URL}/assessment/sessions/{{id}}/result`

export const AUTH_URL = '/auth'
export const RESET_NEW_PASSWORD_URL = `${AUTH_URL}/reset_password`
export const GENERATE_NEW_PASSWORD_URL = `${AUTH_URL}/generate_new_password_email`
export const SIGN_IN_URL = `${AUTH_URL}/signin`
export const LOGOUT_URL = `${AUTH_URL}/logout`

// ui
export const LOGGIN_SETTINGS_URL = `${AUTH_URL}/login_settings`
