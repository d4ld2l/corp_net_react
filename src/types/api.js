import type { AxiosPromise } from 'axios'

import type {
  BirthDatesNearestRaw,
  CandidateRaw,
  CandidateWithIncludesRaw,
  EmployeesRaw,
  EmployeeRaw,
  EventsRaw,
  EventRaw,
  EventTypesRaw,
  ParticipantsRaw,
  UserRaw,
  MilestonesGroupRaw,
  MilestoneTemplateRaw,
  NewRaw,
  NewsRaw,
  NewsCategoriesRaw,
  ProfileRaw,
  ProfileRoleRaw,
  RecruiterRoleInfoRaw,
  SurveysRaw,
  SurveyRaw,
  TagsRaw,
  VacancyRaw,
  VacanciesRaw,
  CandidateVacancyRaw,
  ResumeRaw,
  AdditionalContactRaw,
  ResumeWorkExperienceRaw,
  ResumeRecommendationRaw,
  LanguageSkillRaw,
  LanguageRaw,
  LanguageLevelRaw,
  ResumeEducationRaw,
  EducationLevelRaw,
  CandidatePermittedAttrsRaw,
  ResumeSourceRaw,
  ServicesRaw,
  ServiceRaw,
  BidsRaw,
  BidRaw,
} from './raws'

import type { DepartmentsRaw } from '../presenters/departments'

export type Api = {
  birthdates: {
    nearest: () => AxiosPromise<BirthDatesNearestRaw>,
  },
  candidates: {
    all: () => AxiosPromise<Array<CandidateWithIncludesRaw>>,
    select: (params: string) => AxiosPromise<Array<CandidateWithIncludesRaw>>,
    search: (params: CandidatePermittedAttrsRaw) => AxiosPromise<Array<CandidateWithIncludesRaw>>,
    advancedSearch: (
      params: CandidatePermittedAttrsRaw
    ) => AxiosPromise<Array<CandidateWithIncludesRaw>>,
    update: (
      id: number,
      params: CandidatePermittedAttrsRaw
    ) => AxiosPromise<CandidateWithIncludesRaw | { success: false }>,
    findById: (id: number) => AxiosPromise<CandidateWithIncludesRaw>,
    whereGroupId: (id: number) => AxiosPromise<Array<CandidateWithIncludesRaw>>,
    create: (
      params: CandidatePermittedAttrsRaw
    ) => AxiosPromise<CandidateWithIncludesRaw | { success: false }>,
    parsing: (params: {
      source: string,
      file: string,
    }) => AxiosPromise<CandidateWithIncludesRaw | { success: false }>,
  },
  news: {
    create: (params: Object) => AxiosPromise<NewRaw>,
    all: () => AxiosPromise<NewsRaw>,
    where: (params: { params: * }) => AxiosPromise<NewsRaw>,
    findById: (id: number) => AxiosPromise<NewRaw>,
    categories: () => AxiosPromise<NewsCategoriesRaw>,
  },
  employees: {
    findById: (id: number) => AxiosPromise<EmployeeRaw>,
    search: (params: { params: * }) => AxiosPromise<EmployeesRaw>,
    all: () => AxiosPromise<EmployeesRaw>,
  },
  events: {
    all: () => AxiosPromise<EventsRaw>,
    findById: (id: number) => AxiosPromise<EventRaw>,
    types: () => AxiosPromise<EventTypesRaw>,
    create: (params: Object) => AxiosPromise<EventRaw>,
    update: (id: number, params: Object) => AxiosPromise<EventRaw>,
    delete: (id: number) => AxiosPromise<{}>,
    searchParticipants: (q: string) => AxiosPromise<ParticipantsRaw>,
  },
  auth: {
    signin: (params: Object) => AxiosPromise<UserRaw>,
    logout: () => AxiosPromise<UserRaw>,
  },
  profiles: {
    me: () => AxiosPromise<ProfileRaw>,
  },
  permissions: {
    my: () => AxiosPromise<ProfileRoleRaw>,
  },
  surveys: {
    all: () => AxiosPromise<SurveysRaw>,
    findById: (id: number) => AxiosPromise<SurveyRaw>,
    search: (q: string) => AxiosPromise<SurveysRaw>,
    save: (params: Object) => AxiosPromise<SurveyRaw>,
  },
  users_admin: {
    where: (params: { params: * }) => AxiosPromise<RecruiterRoleInfoRaw>,
  },
  tags: {
    all: () => AxiosPromise<TagsRaw>,
  },
  vacancies: {
    all: () => AxiosPromise<VacanciesRaw>,
    myVacancies: () => AxiosPromise<VacanciesRaw>,
    withStages: (stage: string) => AxiosPromise<VacanciesRaw>,
    changeState: (id: number, state: string) => AxiosPromise<VacancyRaw>,
    findBy: (params: { vacancy: * }) => AxiosPromise<VacancyRaw>,
    update: (id: number, params: { vacancy: * }) => AxiosPromise<VacancyRaw>,
    findById: (id: number) => AxiosPromise<VacancyRaw>,
  },
  vacancy_stage_group: () => AxiosPromise<MilestonesGroupRaw>,
  template_stages: () => AxiosPromise<MilestoneTemplateRaw>,
  resume_sources: () => AxiosPromise<Array<ResumeSourceRaw>>,
  languages: () => AxiosPromise<Array<LanguageRaw>>,
  languagesLevel: () => AxiosPromise<Array<LanguageLevelRaw>>,
  educationLevel: () => AxiosPromise<Array<EducationLevelRaw>>,
  departments: {
    tree: () => AxiosPromise<DepartmentsRaw>,
  },
  services: {
    all: () => AxiosPromise<ServicesRaw>,
    find: () => AxiosPromise<ServiceRaw>,
  },
  bids: {
    all: () => AxiosPromise<BidsRaw>,
    find: () => AxiosPromise<BidRaw>,
  },
}
