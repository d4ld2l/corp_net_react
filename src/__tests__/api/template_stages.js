import api from '../../api'

describe('API Template Stages', () => {
  it('should fetch template stages', async () => {
    const response = await api.template_stages()
    expect(response.data).toMatchSnapshot()
  })
})
