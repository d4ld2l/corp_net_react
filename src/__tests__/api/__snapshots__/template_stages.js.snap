// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`API Template Stages should fetch template stages 1`] = `
Array [
  Object {
    "created_at": "2017-08-09T08:26:22.001Z",
    "id": 1,
    "name": "Базовый",
    "updated_at": "2017-08-09T08:26:22.001Z",
    "vacancy_stages": Array [
      Object {
        "can_create_left": false,
        "can_create_right": false,
        "created_at": "2017-08-15T08:01:21.272Z",
        "editable": false,
        "evaluation_of_candidate": false,
        "group_name": null,
        "id": 113,
        "must_be_last": false,
        "name": "Добавлены",
        "need_notification": false,
        "position": 0,
        "template_stage_id": 1,
        "type_of_rating": "passing",
        "updated_at": "2017-08-15T08:01:21.314Z",
        "vacancy_id": null,
        "vacancy_stage_group": Object {
          "color": "#ffe9bb",
          "created_at": "2017-08-10T15:03:55.026Z",
          "id": 5,
          "label": "Новые",
          "updated_at": "2017-08-10T15:03:55.026Z",
          "value": "new",
        },
        "vacancy_stage_group_id": 5,
      },
      Object {
        "can_create_left": false,
        "can_create_right": true,
        "created_at": "2017-08-15T08:01:21.278Z",
        "editable": false,
        "evaluation_of_candidate": false,
        "group_name": null,
        "id": 114,
        "must_be_last": false,
        "name": "Откликнулись",
        "need_notification": false,
        "position": 1,
        "template_stage_id": 1,
        "type_of_rating": "passing",
        "updated_at": "2017-08-15T08:01:21.317Z",
        "vacancy_id": null,
        "vacancy_stage_group": Object {
          "color": "#ffe9bb",
          "created_at": "2017-08-10T15:03:55.026Z",
          "id": 5,
          "label": "Новые",
          "updated_at": "2017-08-10T15:03:55.026Z",
          "value": "new",
        },
        "vacancy_stage_group_id": 5,
      },
      Object {
        "can_create_left": true,
        "can_create_right": false,
        "created_at": "2017-08-15T08:01:21.283Z",
        "editable": false,
        "evaluation_of_candidate": false,
        "group_name": null,
        "id": 115,
        "must_be_last": false,
        "name": "Принят",
        "need_notification": true,
        "position": 2,
        "template_stage_id": 1,
        "type_of_rating": "ten_point_scale",
        "updated_at": "2017-08-15T08:01:21.319Z",
        "vacancy_id": null,
        "vacancy_stage_group": Object {
          "color": "#ff0",
          "created_at": "2017-08-10T15:03:55.045Z",
          "id": 10,
          "label": "Принят",
          "updated_at": "2017-08-10T15:03:55.045Z",
          "value": "accepted",
        },
        "vacancy_stage_group_id": 10,
      },
      Object {
        "can_create_left": false,
        "can_create_right": false,
        "created_at": "2017-08-15T08:01:21.287Z",
        "editable": false,
        "evaluation_of_candidate": false,
        "group_name": null,
        "id": 116,
        "must_be_last": true,
        "name": "Отказ",
        "need_notification": false,
        "position": null,
        "template_stage_id": 1,
        "type_of_rating": "passing",
        "updated_at": "2017-08-15T08:01:21.320Z",
        "vacancy_id": null,
        "vacancy_stage_group": Object {
          "color": "#0aa",
          "created_at": "2017-08-10T15:03:55.046Z",
          "id": 11,
          "label": "Отказ",
          "updated_at": "2017-08-10T15:03:55.046Z",
          "value": "rejected",
        },
        "vacancy_stage_group_id": 11,
      },
      Object {
        "can_create_left": false,
        "can_create_right": false,
        "created_at": "2017-08-15T08:01:21.291Z",
        "editable": false,
        "evaluation_of_candidate": false,
        "group_name": null,
        "id": 117,
        "must_be_last": true,
        "name": "Архив",
        "need_notification": false,
        "position": null,
        "template_stage_id": 1,
        "type_of_rating": "passing",
        "updated_at": "2017-08-15T16:52:04.096Z",
        "vacancy_id": null,
        "vacancy_stage_group": Object {
          "color": "#aaa",
          "created_at": "2017-08-10T15:03:55.047Z",
          "id": 12,
          "label": "Архив",
          "updated_at": "2017-08-10T15:03:55.047Z",
          "value": "archived",
        },
        "vacancy_stage_group_id": 12,
      },
    ],
  },
  Object {
    "created_at": "2017-08-15T08:01:40.309Z",
    "id": 2,
    "name": "Стандартный",
    "updated_at": "2017-08-15T08:01:40.309Z",
    "vacancy_stages": Array [
      Object {
        "can_create_left": false,
        "can_create_right": false,
        "created_at": "2017-08-15T08:10:09.454Z",
        "editable": false,
        "evaluation_of_candidate": false,
        "group_name": null,
        "id": 127,
        "must_be_last": false,
        "name": "Добавлены",
        "need_notification": false,
        "position": 0,
        "template_stage_id": 2,
        "type_of_rating": null,
        "updated_at": "2017-08-15T08:10:09.458Z",
        "vacancy_id": null,
        "vacancy_stage_group": Object {
          "color": "#ffe9bb",
          "created_at": "2017-08-10T15:03:55.026Z",
          "id": 5,
          "label": "Новые",
          "updated_at": "2017-08-10T15:03:55.026Z",
          "value": "new",
        },
        "vacancy_stage_group_id": 5,
      },
      Object {
        "can_create_left": false,
        "can_create_right": true,
        "created_at": "2017-08-15T08:10:14.726Z",
        "editable": false,
        "evaluation_of_candidate": false,
        "group_name": null,
        "id": 128,
        "must_be_last": false,
        "name": "Откликнулись",
        "need_notification": false,
        "position": 1,
        "template_stage_id": 2,
        "type_of_rating": null,
        "updated_at": "2017-08-15T08:10:14.729Z",
        "vacancy_id": null,
        "vacancy_stage_group": Object {
          "color": "#ffe9bb",
          "created_at": "2017-08-10T15:03:55.026Z",
          "id": 5,
          "label": "Новые",
          "updated_at": "2017-08-10T15:03:55.026Z",
          "value": "new",
        },
        "vacancy_stage_group_id": 5,
      },
      Object {
        "can_create_left": true,
        "can_create_right": true,
        "created_at": "2017-08-15T08:10:19.710Z",
        "editable": true,
        "evaluation_of_candidate": true,
        "group_name": null,
        "id": 129,
        "must_be_last": false,
        "name": "Скрининг рекрутера",
        "need_notification": false,
        "position": 2,
        "template_stage_id": 2,
        "type_of_rating": "passing",
        "updated_at": "2017-08-15T08:10:19.714Z",
        "vacancy_id": null,
        "vacancy_stage_group": Object {
          "color": "#f00",
          "created_at": "2017-08-10T15:03:55.039Z",
          "id": 6,
          "label": "Скрининг",
          "updated_at": "2017-08-10T15:03:55.039Z",
          "value": "screening",
        },
        "vacancy_stage_group_id": 6,
      },
      Object {
        "can_create_left": true,
        "can_create_right": true,
        "created_at": "2017-08-15T08:10:24.614Z",
        "editable": true,
        "evaluation_of_candidate": true,
        "group_name": null,
        "id": 130,
        "must_be_last": false,
        "name": "Скрининг менеджера",
        "need_notification": true,
        "position": 3,
        "template_stage_id": 2,
        "type_of_rating": "passing",
        "updated_at": "2017-08-15T08:10:24.619Z",
        "vacancy_id": null,
        "vacancy_stage_group": Object {
          "color": "#f00",
          "created_at": "2017-08-10T15:03:55.039Z",
          "id": 6,
          "label": "Скрининг",
          "updated_at": "2017-08-10T15:03:55.039Z",
          "value": "screening",
        },
        "vacancy_stage_group_id": 6,
      },
      Object {
        "can_create_left": true,
        "can_create_right": true,
        "created_at": "2017-08-15T08:10:29.286Z",
        "editable": true,
        "evaluation_of_candidate": true,
        "group_name": null,
        "id": 131,
        "must_be_last": false,
        "name": "Интервью рекрутера",
        "need_notification": false,
        "position": 4,
        "template_stage_id": 2,
        "type_of_rating": "passing",
        "updated_at": "2017-08-15T08:10:29.291Z",
        "vacancy_id": null,
        "vacancy_stage_group": Object {
          "color": "#f0f",
          "created_at": "2017-08-10T15:03:55.041Z",
          "id": 7,
          "label": "Интервью",
          "updated_at": "2017-08-10T15:03:55.041Z",
          "value": "interview",
        },
        "vacancy_stage_group_id": 7,
      },
      Object {
        "can_create_left": true,
        "can_create_right": true,
        "created_at": "2017-08-15T08:10:34.287Z",
        "editable": true,
        "evaluation_of_candidate": true,
        "group_name": null,
        "id": 132,
        "must_be_last": false,
        "name": "Интервью менеджера",
        "need_notification": true,
        "position": 5,
        "template_stage_id": 2,
        "type_of_rating": "passing",
        "updated_at": "2017-08-15T08:10:34.290Z",
        "vacancy_id": null,
        "vacancy_stage_group": Object {
          "color": "#f0f",
          "created_at": "2017-08-10T15:03:55.041Z",
          "id": 7,
          "label": "Интервью",
          "updated_at": "2017-08-10T15:03:55.041Z",
          "value": "interview",
        },
        "vacancy_stage_group_id": 7,
      },
      Object {
        "can_create_left": true,
        "can_create_right": true,
        "created_at": "2017-08-15T08:10:38.715Z",
        "editable": true,
        "evaluation_of_candidate": false,
        "group_name": null,
        "id": 133,
        "must_be_last": false,
        "name": "Оффер",
        "need_notification": true,
        "position": 6,
        "template_stage_id": 2,
        "type_of_rating": null,
        "updated_at": "2017-08-15T08:10:38.719Z",
        "vacancy_id": null,
        "vacancy_stage_group": Object {
          "color": "#00f",
          "created_at": "2017-08-10T15:03:55.043Z",
          "id": 9,
          "label": "Оффер",
          "updated_at": "2017-08-10T15:03:55.043Z",
          "value": "offer",
        },
        "vacancy_stage_group_id": 9,
      },
      Object {
        "can_create_left": true,
        "can_create_right": false,
        "created_at": "2017-08-15T08:10:43.751Z",
        "editable": false,
        "evaluation_of_candidate": false,
        "group_name": null,
        "id": 134,
        "must_be_last": false,
        "name": "Принят",
        "need_notification": false,
        "position": 7,
        "template_stage_id": 2,
        "type_of_rating": null,
        "updated_at": "2017-08-15T08:10:43.755Z",
        "vacancy_id": null,
        "vacancy_stage_group": Object {
          "color": "#ff0",
          "created_at": "2017-08-10T15:03:55.045Z",
          "id": 10,
          "label": "Принят",
          "updated_at": "2017-08-10T15:03:55.045Z",
          "value": "accepted",
        },
        "vacancy_stage_group_id": 10,
      },
      Object {
        "can_create_left": false,
        "can_create_right": false,
        "created_at": "2017-08-15T08:10:47.592Z",
        "editable": false,
        "evaluation_of_candidate": false,
        "group_name": null,
        "id": 135,
        "must_be_last": true,
        "name": "Отказ",
        "need_notification": false,
        "position": 8,
        "template_stage_id": 2,
        "type_of_rating": null,
        "updated_at": "2017-08-15T08:10:47.596Z",
        "vacancy_id": null,
        "vacancy_stage_group": Object {
          "color": "#0aa",
          "created_at": "2017-08-10T15:03:55.046Z",
          "id": 11,
          "label": "Отказ",
          "updated_at": "2017-08-10T15:03:55.046Z",
          "value": "rejected",
        },
        "vacancy_stage_group_id": 11,
      },
      Object {
        "can_create_left": false,
        "can_create_right": false,
        "created_at": "2017-08-15T08:10:52.150Z",
        "editable": false,
        "evaluation_of_candidate": false,
        "group_name": null,
        "id": 136,
        "must_be_last": true,
        "name": "Архив",
        "need_notification": false,
        "position": null,
        "template_stage_id": 2,
        "type_of_rating": null,
        "updated_at": "2017-08-15T16:52:04.100Z",
        "vacancy_id": null,
        "vacancy_stage_group": Object {
          "color": "#aaa",
          "created_at": "2017-08-10T15:03:55.047Z",
          "id": 12,
          "label": "Архив",
          "updated_at": "2017-08-10T15:03:55.047Z",
          "value": "archived",
        },
        "vacancy_stage_group_id": 12,
      },
    ],
  },
]
`;
